src/main/java/pl/cleankod/exchange/core/service/AccountService.java

```diff
@@ -0,0 +1,11 @@
+package pl.cleankod.exchange.core.service;
+
+import pl.cleankod.exchange.core.domain.Account;
+
+public interface AccountService {
```

Comment:
What is the point of extracting an interface that'd have only one implementation, and putting it in the same package?